sdks/wasm/driver.c

```diff
@@ -761,6 +761,18 @@ mono_wasm_get_obj_type (MonoObject *obj)
 	}
 }
 
+EMSCRIPTEN_KEEPALIVE int
+mono_wasm_is_simple_array (MonoObject *obj)
+{
```

Comment:
You can do if (a is Array arr) { if (arr.Rank == 1 && a.GetLowerdBound (0) == 0) ... }

Reply:
@vargaz I can do `if (a is Array arr)` in a `.c` program?  That looks more like c-sharp.

Reply:
I mean the calling c# code can do that instead of adding an icall.

Reply:
Address icall in recent commit.